variables.tf

```diff
@@ -332,6 +332,7 @@ variable "firelens" {
     init_config_files            = optional(list(string), [])
     log_level                    = optional(string, "info")
     opensearch_host              = optional(string, "")
+    aws_sigv4_region             = optional(string)
```

Comment:
Maybe call this `aws_region` like in the [docs](https://docs.fluentbit.io/manual/pipeline/outputs/opensearch)?

> Specify the AWS region for Amazon OpenSearch Service
	
We could also add `aws_auth` (defaulting to `On`) which is described as

> Enable AWS Sigv4 Authentication for Amazon OpenSearch Service